### PR TITLE
vim install of pathogen and plugins as described in README.md

### PR DESCRIPTION
**Pull request by dspolleke**
https://github.com/geerlingguy/mac-dev-playbook#configuration-to-be-added

### CODE REVIEW

File: default.config.yml [hunk 1]
```diff
@@ -12,7 +13,7 @@ sudoers_custom_config: ''
 #   # Allow users in admin group to use sudo with no password.
 #   %admin ALL=(ALL) NOPASSWD: ALL
 
-dotfiles_repo: https://github.com/geerlingguy/dotfiles.git
+dotfiles_repo: https://github.com/dspolleke/dotfiles.git
```
Comment (StephenBrown2) [inline default.config.yml:16]: This should not change, at least not in this repo.

File: default.config.yml [hunk 2]
```diff
@@ -85,7 +86,7 @@ composer_packages: []
 #   state: present # present/absent, default: present
 #   version: "^8.1" # default: N/A
 gem_packages: []
-# - name: bundler
+#  - name: showoff
```
Comment (StephenBrown2) [inline default.config.yml:89]: Space mistake:
```suggestion
# - name: showoff
```

File: inventory [hunk 3]
```diff
@@ -1,2 +1,3 @@
 [all]
 127.0.0.1  ansible_connection=local
+#192.168.2.49
```
Comment (StephenBrown2) [inline inventory:3]: A local addition, should be removed

File: default.config.yml (hunk 1 above)
Comment (dspolleke) [reply default.config.yml]: leakage of my fork into upstream. Fixed

File: default.config.yml (hunk 2 above)
Comment (dspolleke) [reply default.config.yml]: leakage of my fork into upstream. Fixed

File: inventory (hunk 3 above)
Comment (dspolleke) [reply inventory]: leakage of my fork into upstream. Fixed

**Review by geerlingguy — CHANGES_REQUESTED** [review]
Can you rebase with just the relevant changes?

**Comment by dspolleke** [comment]
new pr comming soon

**Pull request closed without merging**